### ISSUE TITLE
Ventilation for Room 10

### ISSUE THREAD

**Issue by cryptogoth** [invisible-college/arcology-website#44]
Two 4" holes need to be drilled in the temporary wall for Room 10, as far apart as possible close to ceiling to circulate air. Use plastic sheeting so dust doesn't get on Paul's bed, and vacuum up when done. Don't let the circular cutouts fall into the temporary wall, in between the sheetrock.

+5V and ground need to be run on the 20 AWG wire (white and black) between the two existing fans to an ATX power supply mounted near a power outlet onto a temporary wall.

Waiting on quiet fans arriving via Amazon Locker on Thursday.
The two fans should be mounted to blow in opposite directions so air circulates through the room.

Eventually, the fans will have laundry duct feeding into and out of them, to bring in heated / cooled air and oxygen, and take out carbon dioxide, while not conducting sound.

**Comment by cryptogoth** [comment]
Both intake and outtake fans are installed. More powerful fans for improved airflow and laundry duct heating from miners can wait until another ticket.
